src/Carousel.jsx

```diff
@@ -0,0 +1,100 @@
+/*
+heart icon in upper right corner to favorite- clicking fills in color
+below heart icon is expand image icon. containers for both are 36x36
+clicking on the image brings it out into a light box
+next/prev buttons are on far left & right side of image, vertically centered
+  circular white buttons with a drop shadow, arrows are black single chevrons (svg can be downloaded from etsy)
+  chevron box is 24x24px, button is 48x48
+  drop shadow slightly darkens on hover
+main image ~ 625px wide
+*/
+
+import React from 'react';
+import './styles/styles.css';
+
+class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      favorite: false
+    };
+  }
+
+  favoriteItem() {
+    if (this.state.favorite) {
+      document.getElementById('fav-heart-icon').setAttribute('style', 'display: none;');
+      this.setState({favorite: false});
+    } else {
+      document.getElementById('fav-heart-icon').setAttribute('style', 'display: block;');
+      this.setState({favorite: true});
+    }
+  }
```

Comment:
Could look into doing this by passing in the target from the click event. It would allow you to take out the document references. Having them here may make later testing difficult.